### PR TITLE
Fix NFT search bar position and metadata spacing

### PR DESCRIPTION
**Pull request by michalszorad**

### CODE REVIEW

File: src/Nfts/NftsNavigator.tsx [hunk 1]
```diff
@@ -40,7 +41,11 @@ const Routes = () => {
           title: strings.title,
           headerTitleAlign: 'center',
           header: searchVisible
-            ? () => <SearchHeader placeholder={strings.search} onClose={handleSearchClose} />
+            ? () => (
+                <SafeAreaView edges={['left', 'top', 'right']}>
+                  <SearchHeader placeholder={strings.search} onClose={handleSearchClose} />
+                </SafeAreaView>
+              )
```
Comment (michalszorad) [inline src/Nfts/NftsNavigator.tsx]: Fixes the following
Screenshot 2023-02-28 at 11 45 20

File: src/NftDetails/NftDetails.tsx [hunk 2]
```diff
@@ -144,6 +144,8 @@ const NftMetadataPanel = ({nft}: {nft: YoroiNft}) => {
           </View>
         </Link>
       </MetadataRow>
+
+      <Spacer height={24} />
```
Comment (michalszorad) [inline src/NftDetails/NftDetails.tsx:149]: Adds extra space at the bottom to make the divider visible
image (1)

**Review by banklesss — APPROVED** [review]

**Comment by wolverineks** [comment]
is there a jira ticket for this?

**Comment by michalszorad** [comment]
@wolverineks only a discussion on Slack https://emurgo.slack.com/archives/GJX3N9ZPH/p1677502398056489

**Review by stackchain — APPROVED** [review]

**Pull request merged**